Js oData common: Serialize entity-to-json capability (#1288)
SAP/cloud-sdk: docs-js/features/odata/common/entity/serialize-entity.mdx

> Other rare cases will bring you to the situation where you want to serialize your Entity back to the oData structure you did receive before.
> If you need to transform it from your SAP Cloud SDK representation of an entity back to a oData representation,
> you can use the `executeRaw()` functionality directly within your original request or
> you can serialize the SDK representation / entity by using the `serializeEntity` function.
> This function is part of the common oData service and has to be imported from the SAP Cloud SDK `odata-common` package.

Reviewer: ```suggestion
If you need to transform entities from the typed SAP Cloud SDK representation to their OData representation, use the `serializeEntity()` function of the `entitySerializer` from the `@sap-cloud-sdk/odata-common` package:
```

```ts
import {
  entitySerializer
} from '@sap-cloud-sdk/odata-common';
import { businessPartnerService } from './generated/business-partner-service';

const { businessPartnerApi } = businessPartnerService();

const deserializedEntity = businessPartnerApi
  .entityBuilder()
  .firstName('Peter')
  .lastName('Pan')
  .toBusinessPartnerAddress([
    businessPartnerAddressApi.entityBuilder().country('Neverland').build()
  ])
  .build();

const serializedEntity = entitySerializer(
  businessPartnerApi.deSerializers
).serializeEntity(deserializedEntity, businessPartnerApi);

console.log(serializedEntity); 

> => {
>   FirstName: 'Peter',
>   LastName: 'Pan',
>   to_BusinessPartnerAddress: [
>     {
>       Country: 'Neverland'
>     }
>   ]
> }

Reviewer: ```suggestion
//{
//  FirstName: 'Peter',
//  LastName: 'Pan',
//  to_BusinessPartnerAddress: [
//   {
//      Country: 'Neverland'
//    }
//  ]
//}
```
